providers/src/airflow/providers/google/provider.yaml

```
@@ -170,6 +170,7 @@ dependencies:
   # A transient dependency of google-cloud-bigquery-datatransfer, but we
   # further constrain it since older versions are buggy.
   - proto-plus>=1.19.6
+  - pyarrow>=14.0.1
```

Please add a comment in the code explaining why this is needed. This is useful for when we evaluate the dependencies.
```suggestion
  # place explnation
  - pyarrow>=14.0.1
```

---

@eladkal i'm only making the existing dependency explicit, I added a comment explaining what it is being used for from my understanding (i havent seen this used anywhere else in the provider), but i'm not the author of the code that actually uses it :)

---

Yeah. The comment look very good.